src/pages/Login.tsx

```diff
@@ -4,43 +4,52 @@ import { LoginForm } from "@/components/login/LoginForm";
 import styled from "@emotion/styled";
 import { useNavigate, useLocation } from "react-router-dom";
 import { useAuth } from "@/contexts/AuthContext";
-
-const Container = styled.div`
-  width: 100%;
-  min-height: 90vh;
-  background-color: ${({ theme }) => theme.colors.backgroundDefault};
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-`;
-
-const Logo = styled.img`
-  width: 9rem;
-  margin-bottom: ${({ theme }) => theme.spacing.spacing7};
-`;
+import { ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css"; 
 
 const LoginPage = () => {
   const navigate = useNavigate();
   const location = useLocation();
-  const { login } = useAuth(); 
+  const { login } = useAuth();
 
   const redirectPath = location.state?.from?.pathname || "/";
 
-  const handleLoginSuccess = (email: string) => {
-    login({ id: email, email }); 
-    navigate(redirectPath, { replace: true }); 
+  const handleLoginSuccess = (email: string, token: string) => {
+    login({ id: email, email }, token);
+    navigate(redirectPath, { replace: true });
   };
 
   return (
     <PageContainer>
       <Navigation />
       <Container>
         <Logo src="/assets/kakao_logo.svg" alt="카카오 로고" />
-        <LoginForm onLoginSuccess={handleLoginSuccess} />
+        <LoginForm onLoginSuccess={handleLoginSuccess} /> {/* ✅ 이쪽도 수정됨 */}
+        <ToastContainer
+          position="bottom-center"
+          autoClose={3000}
```

Comment:
numeric separotr를 사용해 볼 수 있을거 같네요
```suggestion
          autoClose={3_000}
```

Reply:
아래처럼 표기하면 직관적으로 몇 초인지 쉽게 알기 좋을거 같아요
```suggestion
          autoClose={3 * 1_000}
```